pype_schema/parse_json.py

```diff
@@ -220,6 +221,120 @@ def merge_network(self, old_network, inplace=False):
             self.network_obj = old_network
         return old_network
 
+    def extend_node(
+        self,
+        new_network,
+        target_node_id,
+        connections_path,
+        inplace=False,
+        verbose=False,
+    ):
+        """
+        Incoporates subnetwork (i.e. the `new_network`) into a node
```

Comment:
Maybe it is just the docstring, but the difference between `merge_network` and `extend_node` is not clear to me